lib/Conversion/TosaToEmitC/TosaToEmitC.cpp

```diff
@@ -118,6 +118,33 @@ class CallOpConversion : public OpConversionPattern<SrcOp> {
   bool explicitOperandTypes;
 };
 
+/// Convert `tosa.concat` into an `emitc.call` operation.
+class ConcatOpConversion : public OpConversionPattern<tosa::ConcatOp> {
+
+public:
+  ConcatOpConversion(MLIRContext *ctx) : OpConversionPattern(ctx) {}
+
+private:
+  LogicalResult
+  matchAndRewrite(tosa::ConcatOp concatOp, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    StringRef funcName = "emitc::tosa::concat";
```

Comment:
Consider using StringLiteral.

Reply:
Since we use `StringRef funcName` at many places in our code you could leave as is. I will refactor the code anyway ad using `StringLiteral` is the better way.